variables.tf

```diff
@@ -49,7 +49,7 @@ variable "override_origin_bucket_policy" {
 variable "origin_bucket" {
   type        = string
   default     = ""
-  description = "Origin S3 bucket name"
+  description = "Use an existing origin S3 bucket name"
```

Comment:
Name of an existing S3 bucket to use as the origin.

Reply:
Good comment. I updated the description with that and more.